packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,34 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  @include terra-consumer-theme-button-default(#64696c, #1c1f21, #383f42);
+
+  /* stylelint-disable selector-class-pattern */
+  .terra-consumer-theme-concept1 {
```

Comment:
unfortunately no. Our current usage is each feature page of the application gets a concept color, and concepts can be re-used across features (it was really fun when the classes used to be events, weight, steps, etc.. and we got to find replace them all to concept#). 
You can also use multiple concepts per feature sometimes (the health assessment has each section using a different concept)
There is also no underlying correlation between 1 concept and a type of feature.

We debated at one point making up abritrary fun names like `Concept 1 is red so find a red random pokemon name, etc` but we ended up being lame and going with concept#. We are open to suggestions, especially if UX has any guidance on how they will be used 
cc @neilpfeiffer